forgerock-auth/src/main/java/org/forgerock/android/auth/AccountSingleSignOnManager.java

```diff
@@ -55,7 +55,7 @@
     private Account account;
 
     @Builder
-    public AccountSingleSignOnManager(@NonNull Context context, Encryptor encryptor) throws Exception {
+    AccountSingleSignOnManager(@NonNull Context context, @NonNull String accountName, Encryptor encryptor) throws Exception {
```

Comment:
Isn't that a breaking change? (changing the access modifier and the method signature...) Hopefully customers are not using it in someway...

Reply:
This is considered internal class, and it is using  builder pattern to build the object, the constructor never been use externally.